ChatApp/ChatView/MessageView.swift

```diff
@@ -0,0 +1,192 @@
+            containerView.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: CGFloat(Constants.containerViewTrailingAnchor)),
+            containerView.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: CGFloat(Constants.containerViewBottomAnchor))
+        ])
+    }
+    
+    func setupTextDateConstant() {
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            textView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: CGFloat(Constants.textViewLeadingAnchor)),
+            textView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor, constant: CGFloat(Constants.textViewBottomAnchor)),
+            textView.heightAnchor.constraint(equalToConstant: CGFloat(Constants.textViewHeightAnchor)),
+            textView.topAnchor.constraint(equalTo: containerView.topAnchor, constant: CGFloat(Constants.textViewTopAnchor))
+        ])
+    }
+    
+    func setupButtonConstant() {
+        button.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            button.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            button.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: CGFloat(Constants.buttonTrailingAnchor)),
+            button.topAnchor.constraint(equalTo: containerView.topAnchor),
+            button.leadingAnchor.constraint(equalTo: textView.trailingAnchor, constant: CGFloat(Constants.buttonLeadingAnchor))
+        ])
+    }
+    
+    func setDark() {
+        backgroundColor = Constants.backgroundColor
+        containerView.backgroundColor = Constants.darkContainerViewBackgroundColor
+        textView.textColor = .white
+    }
+    
+    func setLight() {
+        backgroundColor = .white
+        containerView.backgroundColor = .white
+        textView.textColor = .black
+    }
+}
+
+extension MessageView {
+    enum Constants {
+        static let collectionViewTopAnchor = 16
+        static let containerViewLeadingAnchor = 16
+        static let containerViewTrailingAnchor = -16
+        static let containerViewBottomAnchor = -16
+        static let textViewLeadingAnchor = 16
+        static let textViewBottomAnchor = -8
+        static let textViewHeightAnchor = 44
+        static let textViewTopAnchor = 8
+        static let buttonTrailingAnchor = -8
+        static let buttonLeadingAnchor = 10
+        static let containerViewLayerBorderWidth = 1
+        static let containerViewLayerCornerRadius = 28
+        static let textViewFont = 16
+        static let containerViewBackgroundColor = UIColor(hex: "F1F1F1")
+        static let conteinerViewLayerBorderColor = UIColor(hex: "9F60FF").cgColor
+        static let textViewTextColor = UIColor(hex: "C7C7C7")
+        static let backgroundColor = UIColor(hex: "160039")
+        static let darkContainerViewBackgroundColor = UIColor(hex: "160039")
```

Comment:
here also you can create separate enums for components to group them as I wrote above